src/projectscene/qml/Audacity/ProjectScene/trackspanel/TracksPanel.qml

```diff
@@ -41,73 +46,99 @@ Item {
         property string currentItemNavigationName: ""
     }
 
-    ColumnLayout {
-        id: contentColumn
+    RowLayout {
 
         anchors.fill: parent
 
-        readonly property int sideMargin: 12
-        spacing: sideMargin
+        TrackEffectsPanel {
```

Comment:
I would suggest using section in name, as it is misleading with our real panels

Reply:
I don't mind, but what is it that you call "panel" ? Those dockable guys ?

Reply:
yes

Reply:
ok